tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -98,9 +98,9 @@ platform_tests/api/test_chassis_fans.py::TestChassisFans::test_get_serial:
 
 platform_tests/api/test_chassis_fans.py::TestChassisFans::test_set_fans_led:
   skip:
-    reason: "Unsupported platform API"
+    reason: "On Cisco 8000 platform the leds belong to the fan_tray and are set through the fan_tray API. Unsupported platform API"
```

Comment:
Can you make this conditional based on the platform that triggered this?  We would not want to see this error message when running on MLNX platform.

Reply:
@gechiang made the change as per your request.